### DISCUSSION TITLE
Is there anyway to disable BitrateTest?

### DISCUSSION THREAD

**Discussion by jothi-prasath** [jellyfin/jellyfin#11451] [Q&A]
Is there anyway to disable BitrateTest?
![image](https://github.com/jellyfin/jellyfin/assets/64248628/6933bdc3-c7e2-41c8-a96c-c44c1ae88b32)

**Comment by TimGels** [comment]
Not that I know of. If someone knows it is possible, please correct me.

**Comment by gnattu** [comment]
Can you elaborate more on the use case? Why you want to disable this?

**Reply by ivanjx** [reply]
it does not work really well with traefik buffering in my case

**Reply by gnattu** [reply]
If you go to Settings->Playback and set the Network Quality (If you are on wan you will have two options) to a fixed value will it still do bitrate test?

**Reply by ivanjx** [reply]
it works. but it will be better if i can set it globally.

**Reply by jothi-prasath** [reply]
@gnattu is there any option to setting in globally from server side
